From f808d024c7a7955b0516fc75daa1b7630825001f Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Wed, 29 Nov 2017 10:02:50 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368-xikp: Update display clock
 frequency for panel

Target pixel clock rate for refresh rate @60 Hz
	= 1341 * 1981 * 60 = 159391260 Hz ~ 159 MHz

Change-Id: Ia018ad3c1a34c48e676f9b63b06e7a2f1cb658f2
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-xikp.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-xikp.dts b/arch/arm64/boot/dts/rockchip/rk3368-xikp.dts
index c36c47256877..8bf14efbdead 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-xikp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-xikp.dts
@@ -779,7 +779,7 @@
 			native-mode = <&timing0>;
 
 			timing0: timing0 {
-				clock-frequency = <145000000>;
+				clock-frequency = <159000000>;
 				hactive = <1200>;
 				vactive = <1920>;
 				hback-porch = <60>;
-- 
2.35.3

